app/src/main/java/org/piramalswasthya/cho/repositories/BenVisitRepo.kt

```
@@ -479,126 +481,150 @@ class BenVisitRepo @Inject constructor(
                                 }
-                                else -> {}
+                            } else {
+                                Log.d("WU", "processUnsyncedPharmacistData: success6")
+                                patientVisitInfoSyncRepo.updatePharmacistDataSyncState(it.patient.patientID, it.patientVisitInfoSync.benVisitNo, SyncState.SYNCED)
                             }
-                        } else {
-                            patientVisitInfoSyncRepo.updatePharmacistDataSyncState(it.patient.patientID, it.patientVisitInfoSync.benVisitNo, SyncState.SYNCED)
-                        }
 
-                    } else { }
+                        } else { }
```

detekt

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyElseBlock)